src/cmd/singularity/cli/sif.go

```diff
@@ -28,8 +28,8 @@ func init() {
 	SifCmd.Flags().SetInterspersed(false)
 	SingularityCmd.AddCommand(SifCmd)
 	// Create
-	SifCmd.AddCommand(SifCreate)
```

Comment:
I think we can graciously remove anything related to the create command here. I agree that it should not be there anymore. There should not be two different path from singularity to create a SIF image and "build" is the right command to accomplish that.